solr/modules/jwt-auth/build.gradle

```diff
@@ -19,7 +19,15 @@ apply plugin: 'java-library'
 
 description = 'JWT / OpenID Connect / OAuth2 authentication plugin'
 
+configurations {
+  constraintsOnly
+  permitTestUnusedDeclared.extendsFrom constraintsOnly
+  implementation.extendsFrom constraintsOnly
+}
```

Comment:
This is a hacky way demonstrated here: https://github.com/gradle-dependency-analyze/gradle-dependency-analyze/issues/108 - I need to go back and add a comment.